cfgov/jinja2/v1/_layouts/base-base.html

```diff
@@ -0,0 +1,245 @@
+<!DOCTYPE html>
+{% if flag_enabled('CFPB_RECRUITING', request) %}
+<!--
+    ============================================================================
+
+    Hey! If you're viewing this, you should probably come work on our Technology
+    & Innovation team. We're always looking for a few great designers,
+    developers, data scientists, and network, infrastructure, privacy and
+    security pros. Keep an eye on our job opportunities at:
+    http://www.consumerfinance.gov/jobs/
+
+    Also, you can see more of our code at https://github.com/cfpb
+
+    And by the way, there’s another hidden message somewhere on the following
+    page: http://www.consumerfinance.gov/jobs/technology-innovation-fellows/.
+    See if you can find it! Hint: picture yourself embedded in our work.
+
+    ============================================================================
+-->
+{% endif %}
+
+<!--[if lt IE 7]>      <html lang="es" class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html lang="es" class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html lang="en" class="no-js lt-ie10 lt-ie9"> <![endif]-->
+<!--[if IE 9]>         <html lang="en" class="no-js lt-ie10"> <![endif]-->
+<!--[if gt IE 8]><!--> <html lang="en" class="no-js"> <!--<![endif]-->
+
+<head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# {% block og_article_prefix %}{% endblock %}">
+
+{#
+    ===========
+    GLOBAL META
+    ===========
+#}
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta charset="utf-8">
+    {% block meta_viewport %}
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    {% endblock meta_viewport %}
+
+{#
+    ==================
+    PAGE-SPECIFIC META
+    ==================
+#}
+
+    <title>
+        {%- block title -%}
+            {%- if page -%}
+                {{ page.seo_title or page.title }} | Consumer Financial Protection Bureau
+            {%- else -%}
+                Consumer Financial Protection Bureau
+            {%- endif %}
+        {%- endblock title -%}
+    </title>
+    <meta name="description"
+          content="
+            {%- block desc -%}
+                {{- page.search_description if page and page.search_description
+                    else 'Our vision is a consumer finance marketplace that works for American consumers, responsible providers, and the economy as a whole.'
+                -}}
+            {%- endblock -%}
```

Comment:
Please update this to `{% endblock desc %}` for consistency with most of the other non-inline `endblock` tags in this template.